public/javascripts/ScrabBag.js

```diff
@@ -0,0 +1,54 @@
+class ScrabBag {
```

Comment:
@jrob8577, submitted changes under same pull request. I am not sure how to implement reduce or map methods on the functions because of the way they are structured. Also, reduce is still confusing to me. That is why I used old or deprecated methods of achieving the same result. As always, your feedback is greatly appreciated. Thank you.